src/routes/SignIn.jsx

```diff
@@ -0,0 +1,42 @@
+import { signInWithEmailAndPassword } from 'firebase/auth';
+import React, { useState } from 'react';
+import { auth } from '../utils/firebase';
+
+// export default function SignIn() {
+const SignIn = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const signIn = (e) => {
+    e.preventDefault();
+    signInWithEmailAndPassword(auth, email, password)
+      .then((userCredential) => {
+        console.log(userCredential);
+        setEmail('');
+        setPassword('');
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+  return (
+    <div className="sign-in-container">
+      <h1>Sign In</h1>
+      <form id="myForm" onSubmit={signIn}>
+        <input
+          type="email"
+          placeholder="Enter your email address"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}></input>
+        <input
+          type="password"
+          placeholder="Enter Your Password"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}></input>
+        <button type="submit">Log In</button>
+      </form>
+    </div>
+  );
+};
+
+export default SignIn;
```

Comment:
I think Login.jsx and SignIn.jsx are pretty overlapping.. and routes folder is for something that we need a new page for it. We can have GoogleAuth component and UserAuth component in components folder and call them in Login.jsx